src/main/java/org/ehcache/sizeof/ObjectGraphWalker.java

```diff
@@ -139,8 +138,8 @@ long walk(VisitorListener visitorListener, Object... root) {
             traversalDebugMessage = null;
         }
         long result = 0;
-        Stack<Object> toVisit = new Stack<Object>();
-        IdentityHashMap<Object, Object> visited = new IdentityHashMap<Object, Object>();
+        Stack<Object> toVisit = new Stack<>();
```

Comment:
I don't think it's officially deprecated but we should probably switch away from Stack to Deque.  ArrayDeque would be the obvious replacement implementation.